### PR TITLE
Persian (Farsi "Fa") Language Support

### PR DESCRIPTION
**Pull request by blacksrc**

### CODE REVIEW

File: fa/validation.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+		"file"    => ":attribute نباید بزرگتر از :max کیلوبایت باشد.",
+		"string"  => ":attribute نباید بیشتر از max کاراکتر باشد.",
+		"array"   => ":attribute نباید بیشتر از :max آیتم باشد.",
+	),
+	"mimes"            => ":attribute باید یکی از فرمت های :values باشد.",
+	"min"              => array(
+		"numeric" => ":attribute نباید کوچکتر از :max باشد.",
+		"file"    => ":attribute نباید کوچکتر از :max کیلوبایت باشد.",
+		"string"  => ":attribute نباید کمتر از max کاراکتر باشد.",
+		"array"   => ":attribute نباید کمتر از :max آیتم باشد.",
```
Comment (mshahrokhi) [inline fa/validation.php]: :max must be :min

File: fa/validation.php [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+		"numeric" => ":attribute نباید بزرگتر از :max باشد.",
+		"file"    => ":attribute نباید بزرگتر از :max کیلوبایت باشد.",
+		"string"  => ":attribute نباید بیشتر از max کاراکتر باشد.",
+		"array"   => ":attribute نباید بیشتر از :max آیتم باشد.",
+	),
+	"mimes"            => ":attribute باید یکی از فرمت های :values باشد.",
+	"min"              => array(
+		"numeric" => ":attribute نباید کوچکتر از :max باشد.",
+		"file"    => ":attribute نباید کوچکتر از :max کیلوبایت باشد.",
+		"string"  => ":attribute نباید کمتر از max کاراکتر باشد.",
```
Comment (mshahrokhi) [inline fa/validation.php]: :max must be :min

**Comment by andrey-helldar** [comment]
@mshahrokhi, lol 😆

image

**Comment by mshahrokhi** [comment]
:))))))

**Pull request merged**